modules/standard/Set.chpl

```diff
@@ -541,7 +541,7 @@ module Set {
     :return: A new set containing the union between `a` and `b`.
     :rtype: `set(?t, ?)`
   */
-  proc |(const ref a: set(?t, ?), const ref b: set(t, ?)): set(t) {
+  operator set.|(const ref a: set(?t, ?), const ref b: set(t, ?)): set(t) {
```

Comment:
Ditto for these set operators. Since both map and set use operator methods, I'm guessing that list is the odd one out here and `==` for list should probably be `list.==`?